optuna/integration/sklearn.py

```diff
@@ -563,6 +563,22 @@ def classes_(self) -> OneDimArrayLikeType:
 
         return self.best_estimator_.classes_
 
+    @property
+    def cv_results_(self) -> Dict[str, Any]:
+        """Return the dictionary has list of CV results of all trials.
+        Returns:
+            A list of CV results of all trials.
+        """
```

Comment:
These lines seem to be duplicated. How about unifying it like the other properties?

```suggestion
        """A dictionary mapping a metric name to a list of Cross-Validation results of all trials."""
```